src/Resources/views/tailwind_2/_header.html.twig

```diff
@@ -52,6 +51,12 @@
                             <p class="whatwedo-utility-bold whitespace-nowrap mb-3 mt-3">{{ 'araise_table.download.info' | trans }}</p>
                             <div>
                                 {% for label,queryParameters in {'araise_table.page' : app.request.query.all, 'araise_table.all' : {'all':1}|merge(app.request.query.all)} %}
+                                    {% if view.definition != table.option('definition') %}
```

Comment:
NIT: has this been tested with a relation content of the same definition?

Reply:
Good catch, this check is unnecessary. I have removed the check and tested it with an entity category with under categories.